server/src/db.js

```diff
@@ -133,3 +144,49 @@ exports.markersForArgs = function(starting, numberOfArgs) {
   }
   return result.join(", ");
 };
+
+function doNothing() {
+}
+
+function initTiming() {
+  if (!logQueryLimit) {
+    return doNothing;
+  }
+  let caller = getCallerPosition(2);
+  if (caller == "skip") {
+    // This happens when getCallerPosition detects we shouldn't time this function call
+    return doNothing;
+  }
+  let start = Date.now();
```

Comment:
I looked at process.hrtime, but it splits the time into two numbers, and seemed complicated to work with. We don't really care about anything that's less than (or even close to) 1ms anyway.